packages/harmony/src/components/button/BaseButton/BaseButton.tsx

```diff
@@ -93,7 +94,7 @@ export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
       })
     }
 
-    const iconCss = {
+    const iconCss = !isStaticIcon && {
```

Comment:
this fixes issue where metamask icon was being slightly overriden by the button icon styles